src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -1510,6 +1511,7 @@ makeEntity e = do
 descriptionModal :: Entity -> EventM Name AppState ()
 descriptionModal e = do
   s <- get
+  resetViewport modalScroll
```

Comment:
Looks like description modals are handled a little differently than other modals. I wanted to reset the viewport in just one place (`openModal`) but that function doesn't really cover description.